Source/Blazorise/Base/BaseInputComponent.razor.cs

```diff
@@ -125,6 +149,20 @@ protected virtual void ReleaseResources()
         }
     }
 
+    /// <summary>
+    /// Initializes parameters that are not going to be changed again.
+    /// </summary>
+    protected void IntializeParameters()
+    {
+        if ( hasInitializedParameters )
+            return;
+
+        // we only need to generate field names on the server side
+        shouldGenerateFieldNames = !OperatingSystem.IsBrowser();
```

Comment:
Should we try/catch instead of trusting the underlying code? Nowadays it seems like Microsoft introduces too many modes/variables... Maybe just to be sure, this doesn't break in some mode for some reason...

Reply:
Again, the same as in .NET project. The `OperatingSystem.IsBrowser()` basically just checks that we are in WASM browser mode. Should be safe. I don't see any exception in their API docs https://learn.microsoft.com/en-us/dotnet/api/system.operatingsystem.isbrowser?view=net-8.0